pkg/idhubmain/persister.go

```diff
@@ -37,3 +38,11 @@ func initHedgehog(db *gorm.DB) {
 		hedgehog.DataVaultItem{},
 	)
 }
+
+func iniJWTClaimPersister(db *gorm.DB, didJWTService *didjwt.Service) *claimsstore.JWTClaimPGPersister {
```

Comment:
mista T in init
![image](https://user-images.githubusercontent.com/61657/74965830-e30a3500-53e3-11ea-8c2c-f494393ec210.png)